packages/file_selector/file_selector_linux/pigeons/messages.dart

```diff
@@ -52,7 +52,7 @@ class PlatformFileChooserOptions {
   final bool? selectMultiple;
 }
 
-@HostApi(dartHostTestHandler: 'TestFileSelectorApi')
```

Comment:
This wasn't being used, it was just copypasta from some other platform, which is why there are no other changes to file_selector_linux.